### PR TITLE
SecureMixLLC.GlassWireSetup version 3.3.499.0

### PR DESCRIPTION
**Pull request by Masamune3210**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/101865)

### CODE REVIEW

**Comment by stephengillie** [comment]
I'm not sure, but I believe the mention of `malware` [might have caused](https://learn.microsoft.com/en-us/windows/package-manager/package/windows-package-manager-policies#12-security) the `Policy-Test-1.2` label to be added.

File: manifests/s/SecureMixLLC/GlassWireSetup/3.3.499.0/SecureMixLLC.GlassWireSetup.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: SecureMixLLC.GlassWireSetup
+PackageVersion: 3.3.499.0
+Installers:
+- InstallerUrl: https://download.glasswire.com/GlassWireSetup.exe?v=3.3.499&user_agent=TW96aWxsYS81LjAgKFdpbmRvd3MgTlQgMTAuMDsgV2luNjQ7IHg2NCkgQXBwbGVXZWJLaXQvNTM3LjM2IChLSFRNTCwgbGlrZSBHZWNrbykgQ2hyb21lLzExMS4wLjAuMCBTYWZhcmkvNTM3LjM2
```
Comment (Trenly) [inline manifests/s/SecureMixLLC/GlassWireSetup/3.3.499.0/SecureMixLLC.GlassWireSetup.installer.yaml:7]:
```suggestion
- InstallerUrl: https://download.glasswire.com/f/c/12ecb72f94a00248ecf82b9824aaa016/GlassWireSetup.exe
```

File: manifests/s/SecureMixLLC/GlassWireSetup/3.3.499.0/SecureMixLLC.GlassWireSetup.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: SecureMixLLC.GlassWireSetup
+PackageVersion: 3.3.499.0
+PackageLocale: en-US
+Publisher: SecureMix LLC
+PackageName: GlassWire
+License: Copyright SecureMix LLC
```
Comment (Trenly) [inline manifests/s/SecureMixLLC/GlassWireSetup/3.3.499.0/SecureMixLLC.GlassWireSetup.locale.en-US.yaml:9]: A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

File: manifests/s/SecureMixLLC/GlassWireSetup/3.3.499.0/SecureMixLLC.GlassWireSetup.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: SecureMixLLC.GlassWireSetup
+PackageVersion: 3.3.499.0
```
Comment (Trenly) [inline manifests/s/SecureMixLLC/GlassWireSetup/3.3.499.0/SecureMixLLC.GlassWireSetup.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 3.3.499
```
Should Match Control Panel

File: manifests/s/SecureMixLLC/GlassWireSetup/3.3.499.0/SecureMixLLC.GlassWireSetup.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: SecureMixLLC.GlassWireSetup
+PackageVersion: 3.3.499.0
```
Comment (Trenly) [inline manifests/s/SecureMixLLC/GlassWireSetup/3.3.499.0/SecureMixLLC.GlassWireSetup.installer.yaml:5]:
```suggestion
PackageVersion: 3.3.499
```
Should Match Control Panel

File: manifests/s/SecureMixLLC/GlassWireSetup/3.3.499.0/SecureMixLLC.GlassWireSetup.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: SecureMixLLC.GlassWireSetup
+PackageVersion: 3.3.499.0
```
Comment (Trenly) [inline manifests/s/SecureMixLLC/GlassWireSetup/3.3.499.0/SecureMixLLC.GlassWireSetup.yaml:5]:
```suggestion
PackageVersion: 3.3.499
```
Should Match Control Panel

File: manifests/s/SecureMixLLC/GlassWireSetup/3.3.499.0/SecureMixLLC.GlassWireSetup.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: SecureMixLLC.GlassWireSetup
```
Comment (Trenly) [inline manifests/s/SecureMixLLC/GlassWireSetup/3.3.499.0/SecureMixLLC.GlassWireSetup.yaml:4]:
```suggestion
PackageIdentifier: SecureMixLLC.GlassWire
```

File: manifests/s/SecureMixLLC/GlassWireSetup/3.3.499.0/SecureMixLLC.GlassWireSetup.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: SecureMixLLC.GlassWireSetup
```
Comment (Trenly) [inline manifests/s/SecureMixLLC/GlassWireSetup/3.3.499.0/SecureMixLLC.GlassWireSetup.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: SecureMixLLC.GlassWire
```

File: manifests/s/SecureMixLLC/GlassWireSetup/3.3.499.0/SecureMixLLC.GlassWireSetup.installer.yaml [hunk 8]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: SecureMixLLC.GlassWireSetup
```
Comment (Trenly) [inline manifests/s/SecureMixLLC/GlassWireSetup/3.3.499.0/SecureMixLLC.GlassWireSetup.installer.yaml:4]:
```suggestion
PackageIdentifier: SecureMixLLC.GlassWire
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Masamune3210** [comment]
....how is it failing validation? All I did was fix the license and commit the code suggestions

**Comment by stephengillie** [comment]
Automatic Validation failed with:
>  Manifest: The manifest file path must match PackageIdentifier and PackageVersion properties from manifest. actual 'manifests\s\SecureMixLLC\GlassWireSetup\3.3.499.0', expected 'manifests\s\SecureMixLLC\GlassWire\3.3.499' 

This is referencing the `.0` removed from the version in the previous commit - the file paths have to be updated to match. I do not know why validation passed the first time and only caught this the second time.

**Comment by Masamune3210** [comment]
If you can tell me how the best way to fix this, I would appreciate it. I have never figured out how to rename files or folders in a pull request, usually just end up closing the commit and opening a new one with all the fixes

**Comment by stephengillie** [comment]
It should be pretty simple - just move the files from folder  `3.3.499.0` to another folder named `3.3.499`, and commit the file move.

**Comment by Masamune3210** [comment]
Oh great, what is it winging about now lol

**Comment by stephengillie** [comment]
>  Manifest: The filename must match the PackageIdentifier and ManifestType properties from the manifest. actual 'SecureMixLLC.GlassWireSetup.yaml', expected 'SecureMixLLC.GlassWire.yaml'

The filename has to match the PackageIdentifier, which doesn't have `Setup`. Didn't catch it before, sorry.

**Comment by Masamune3210** [comment]
I just checked the url, it downloads just fine for me

**Comment by BrandonWanHuanSheng** [comment]
It look like the hash could be mismatch and the setup file version was changed.

**Comment by Masamune3210** [comment]
the validation was canceled?

**Comment by Masamune3210** [comment]
Think the bot might have barfed up on itself lol
image

**Comment by Masamune3210** [comment]
FINALLY!

**Review by stephengillie — APPROVED** [review]

**Pull request merged**